Langs/0804.lang

```diff
@@ -53,7 +64,22 @@ You're running the latest build!=您当前使用的 WhyNotWin11 是最新版本
 Unable to Check for Updates=无法检查更新
 Unable to load release data.=无法获取版本信息。
 Invalid Data Received!=获取到的数据无效!
-Invalid Release Tags Received!=获取的 GitHub Release 标签无效!
-Invalid Release Types Received!=获取的 GitHub Release 类型无效!
-Update Available=有更新版本
+Invalid Release Tags Received!=获取到的 GitHub Release 标签无效!
+Invalid Release Types Received!=获取到的 GitHub Release 类型无效!
+Update Available=有新版本可用
 An Update is Available, would you like to download it?=新版 WhyNotWin11 发布了，您要前往下载吗?
+Done=完成
+Loading WMIC=正在使用 WMIC 读取信息...
+Warning=警告
+[Descriptions]
+Architecture=指 CPU 和操作系统可以一次处理的数据量。如果操作系统为 32 位，则需要格式化系统盘之后才能安装 Windows 11。如果 CPU 为 32 位，则需要更换 CPU。
+Boot=指主板加载 Windows 的方式。如果结果为“Legacy”，则可以在较新的主板中找到 BIOS/UEFI 设置以进行调整。具体请参阅您的主板使用手册。
+CPU Name=指在计算机中安装的 CPU。与系统的兼容性可能会发生变化。如果您使用的是台式电脑，则可以更换 CPU；笔记本电脑一般无法更换。
+CPU Cores=指 CPU 可以同时处理的任务量。如果您使用的是台式电脑，则可以更换 CPU；笔记本电脑一般无法更换。
+CPU Speed=指 CPU 处理任务的速度。如果您使用的是台式电脑，则可以更换 CPU；笔记本电脑一般无法更换。
+DirectX=指显卡支持的 DirectX DDI/功能层版本。与 DirectX 的软件版本不同，“DirectX 12 API”的显卡可能无法通过此检查。
+Disk Type=指磁盘储存数据的方式。如果分区形式不是 GPT，则可以使用 Microsoft 提供的 MBR2GPT 工具进行修复。
+RAM=指计算机中安装的内存大小。可在台式电脑和高端笔记本电脑升级。
+Secure Boot=指主板加载 Windows 的方式。如果未检测到安全启动，则可以在较新的主板中找到 BIOS/UEFI 设置以进行调整。具体请参阅您的主板使用手册。
+Storage=指磁盘中数据可使用的空间量。有些高端笔记本电脑可以更换或升级。
+TPM=指 Windows 使用的安全模块。所有新款 AMD CPU 自带此模块，部分新款 Intel CPU 自带。请检查主板中的 BIOS/UEFI 设置，具体请参阅您的主板使用手册。
```

Comment:
Haven't got the chance to read the original text to the descriptions section yet. I will do that in the coming morning.